### PR TITLE
src: change icons for different dropdown items

### LINKED ISSUE #175: Change icons for different dropdown items
**Actual Behaviour**

Currently, icons for skills, settings, login and logout are same in dropdown menu

**Expected Behaviour**

The icon should be different for different item.

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by AdityaSrivast**
Fixes #175 

#### Changes proposed in this pull request:
- Currently the icons for skills, settings, login and logout are same in dropdown menu. But there should be different icons for different items.

#### Screenshots for the change: 
image
image

### CODE REVIEW

**Comment by AdityaSrivast** [comment]
@hkedia321 @tstreamDOTh Can you please review?

**Comment by AdityaSrivast** [comment]
@jogendra I have made the changes. Can you please review again?

File: src/images/setting.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path fill="none" d="M0 0h20v20H0V0z"/><path d="M15.95 10.78c.03-.25.05-.51.05-.78s-.02-.53-.06-.78l1.69-1.32c.15-.12.19-.34.1-.51l-1.6-2.77c-.1-.18-.31-.24-.49-.18l-1.99.8c-.42-.32-.86-.58-1.35-.78L12 2.34c-.03-.2-.2-.34-.4-.34H8.4c-.2 0-.36.14-.39.34l-.3 2.12c-.49.2-.94.47-1.35.78l-1.99-.8c-.18-.07-.39 0-.49.18l-1.6 2.77c-.1.18-.06.39.1.51l1.69 1.32c-.04.25-.07.52-.07.78s.02.53.06.78L2.37 12.1c-.15.12-.19.34-.1.51l1.6 2.77c.1.18.31.24.49.18l1.99-.8c.42.32.86.58 1.35.78l.3 2.12c.04.2.2.34.4.34h3.2c.2 0 .37-.14.39-.34l.3-2.12c.49-.2.94-.47 1.35-.78l1.99.8c.18.07.39 0 .49-.18l1.6-2.77c.1-.18.06-.39-.1-.51l-1.67-1.32zM10 13c-1.65 0-3-1.35-3-3s1.35-3 3-3 3 1.35 3 3-1.35 3-3 3z"/></svg>
```
[the remark this answers is not in the document]
Comment (AdityaSrivast) [reply src/images/setting.svg]: @samagragupta sorry I don't know what do we mean by EOF :sweat_smile: . Can you please simplify the term

[the remark this answers is not in the document]
Comment (sk9331657) [reply src/images/setting.svg]: @AdityaSrivast  Just add an empty line at the end of the file.

**Comment by AdityaSrivast** [comment]
@samagragupta @sk9331657  made the changes. Can you please review again?

**Pull request merged**